src/applications/disability-benefits/all-claims/submit-transformer.js

```diff
@@ -218,6 +226,13 @@ export function transform(formConfig, form) {
       ),
     );
 
+  const addBackRatedDisabilities = formData =>
+    savedRatedDisabilities
+      ? _.set('ratedDisabilities', savedRatedDisabilities, formData)
+      : formData;
+
+  const filterRatedViewFields = formData => filterViewFields(formData);
```

Comment:
If I remember correctly, `transformForSubmit` runs this already, so this is for `savedRatedDisabilities` only, right? Can we put this in `addBackRatedDisabilities` so we're not running it on all the data twice (and so we don't have to worry about sequencing later)?